content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -228,6 +255,13 @@ Point.
    it displayed by selecting it from the optional columns, accessible from the top right corner of
    the Invoices list view.
 
+A cron runs regularly to check the status of these invoices. It is possible to check the status
+before the cron runs by clicking on :guilabel:`Fetch Peppol invoice status` in the corresponding
+Sales journal on the Accounting dashboard.
```

Comment:
```suggestion
before the cron runs by clicking :guilabel:`Fetch Peppol invoice status` in the corresponding journal on the Accounting dashboard.
```
since you used "click" and not "click on" everywhere else